operator/webhook/test/test_sync.py

```diff
@@ -44,6 +42,28 @@ def test_vol_config_missing_optional_vols_no_fail(full_route):
     assert len(vol_conf.get_volumes()) > 0
     assert len(vol_conf.get_mounts()) > 0
 
+def test_tls_config_missing(full_route):
+    del full_route["spec"]["tls"]
+    vol_conf = VolumeConfig(full_route["spec"])
+
+    assert len(vol_conf.get_volumes()) > 0
+    assert len(vol_conf.get_mounts()) > 0
+
+    print(vol_conf)
```

Comment:
:question: Is this `print` statement still required?

Reply:
Removed